docs/rest_api.md

```diff
@@ -267,7 +267,38 @@ Example Response:
 
 Authorization requirements:
 
-* Administrative access is required
+* Administrative access is required to list all networks
+
+`GET /networks`
+
+List all public networks. (Regular user operation)
+
+Returns a JSON dictionary of dictionaries, where the exterior dictionary is indexed by
+the network name and the value of each key is another dictionary with keys corresponding
+to that network's id and projects
+
+The response must contain the following fields:
+
+* "network", the name of a network
+* "network_id", the id of the network
+* "projects", a list of projects with access to the network or 'None' if network is public
+
+Example Response:
+
+    {
+        "netA": {
+            "network_id": "101",
+            "projects": None
+        },
+        "netB": {
+            "network_id": "102",
+            "projects": None
+        }
+    }
+
+Authorization requirements:
+
+* No special access is required to list all public networks
```

Comment:
This looks like it's been documented twice. I would just factor out the common part (like the URL, sample body of the request if any, description etc) and have responses for both cases. And in the end, under authorization requirements have 2 bullet points listing it. You can look at how I documented the [`show_node` API](https://github.com/CCI-MOC/hil/blob/master/docs/rest_api.md#show_node)